react-relay/index.d.ts

```diff
@@ -128,7 +146,13 @@ declare module "react-relay" {
     }
 
     interface RelayProp {
-        variables: any
-        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void
+        route: { name: string; }; // incomplete, also has params and queries
+        variables: any;
+        pendingVariables?: any;
+        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        forceFetch(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        hasOptimisticUpdate(record?: any): boolean;
+        getPendingTransactions(record?: any): RelayMutationTransaction[];
```

Comment:
huh... you've corrected me on how I thought this functions. I thought the record was optional and one could ask for optimistic updates for the component you're inside of... oh well, now I know!